### ISSUE TITLE
[Accessibility] [Screen reader -Tags]: Table is not implemented properly in ‘Tags’ page. [Sev3][MAS1.3.1]

### ISSUE THREAD

**Issue by Laxmidurga** [microsoft/tsdoc.org-website#33]
**Test Environment:**
OS Build: Windows 11
Version: 21H2 (OS Build 22000.318)
Browser: Edge
Version 96.0.1054.43 (Official build) (64-bit)
URL: https://tsdoc.org/
Screen reader: Narrator

**Repro Steps:**
1.	Open the above URL and login with valid credentials.
2.	TSDoc page will get displayed.
3.	Now navigate to ‘Tags’ link and activate it.
4.	Now, turn on screen reader and press “CapsLock+Space” key to enter scan mode.
5.	Now, use table shortcut keys ‘T’ to navigate to table and use ‘Ctrl+Alt+arrow keys’ to navigate in table, observe the issue.

**Actual:**
Table is not implemented properly in ‘Tags’ page. Title/Summary of the table is not defined and also headers are not defined for the table.

**Expected:**
Table should be implemented properly in ‘Tags’ page. Title/Summary of the table should be defined and also headers should be defined for the table.

**User Impact:**
Assistive technology (like Screen readers) users will face difficulty as he/she may not be able to know the purpose of the table if title and description of table is not available.

**Note:** Issue is observed for the table in ‘@returns’ page

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships

https://user-images.githubusercontent.com/88436727/150644337-f4c1726a-2557-4ac4-a606-fb3b332af633.mp4

**Comment by octogonz** [comment]
This issue has been closed because this Jekyll project has been retired. The documentation content has been migrated to Docusaurus. 

If the problem has not been addressed, please file a new issue under this GitHub project instead:  

https://github.com/microsoft/rushstack-websites
